docs/extensions/engines/spark/jvm-quake.md

```diff
@@ -0,0 +1,77 @@
+<!--
+- Licensed to the Apache Software Foundation (ASF) under one or more
+- contributor license agreements.  See the NOTICE file distributed with
+- this work for additional information regarding copyright ownership.
+- The ASF licenses this file to You under the Apache License, Version 2.0
+- (the "License"); you may not use this file except in compliance with
+- the License.  You may obtain a copy of the License at
+-
+-   http://www.apache.org/licenses/LICENSE-2.0
+-
+- Unless required by applicable law or agreed to in writing, software
+- distributed under the License is distributed on an "AS IS" BASIS,
+- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+- See the License for the specific language governing permissions and
+- limitations under the License.
+-->
+
+# JVM Quake Support
+
+When facing out-of-control memory management in Spark engine, we typically use JVMkill as a remedy by killing the process and generating a heap dump for post-analysis. However, even with jvmkill protection, we may still encounter issues caused by JVM running out of memory, such as repeated execution of Full GC without performing any useful work during the pause time. Since the JVM does not exhaust 100% of resources, JVMkill will not be triggered.
+
+So introducing JVMQuake provides more granular monitoring of GC behavior, enabling early detection of memory management issues and facilitating fast failure.
+
+## Usage
+
+JVM Quake is implemented through Spark plugins
+
+| Spark Version | Supported | Remark |
+|:-------------:|:---------:|:------:|
+|    master     |     √     |   -    |
+|     3.5.x     |     √     |   -    |
+|     3.4.x     |     √     |   -    |
+|     3.3.x     |     √     |   -    |
+|     3.2.x     |     √     |   -    |
+|     3.1.x     |     √     |   -    |
+|     3.0.x     |     √     |   -    |
+|     2.4.x     |     x     |   -    |
```

Comment:
We could remove these versions from the list. As Kyuubi only supports and runs tests on Spark 3.3+.

Reply:
JVM Quake is not only applicable to Kyuubi, but can also be directly applied to other Spark apps

Reply:
How about clarifying it as: 

This plugin technically supports Spark 3.0 onwards, but was only verified with Spark 3.3 to 4.0 in CI.

Reply:
@yoock my last comment is neither get adoption nor rejection, could you re-evaluate it?

Reply:
addClarification document added